### ISSUE TITLE
Network OVNKubernetesHybridOverlayNetwork not found

### ISSUE THREAD

**Issue by tibcoplord** [openshift/windows-machine-config-bootstrapper#210]
After following these instructions (https://github.com/openshift/windows-machine-config-bootstrapper/blob/master/tools/ansible/docs/azure/azure-with-windows-server.md) my cluster appears to be up and running -

```
$ oc get node
NAME                                STATUS   ROLES    AGE   VERSION
ocp447-nffjf-master-0               Ready    master   19h   v1.17.1+f5fb168
ocp447-nffjf-master-1               Ready    master   19h   v1.17.1+f5fb168
ocp447-nffjf-master-2               Ready    master   19h   v1.17.1+f5fb168
ocp447-nffjf-worker-eastus1-959mg   Ready    worker   19h   v1.17.1+f5fb168
ocp447-nffjf-worker-eastus2-pxtzc   Ready    worker   19h   v1.17.1+f5fb168
ocp447-nffjf-worker-eastus3-vtrqr   Ready    worker   19h   v1.17.1+f5fb168
win-worker1                         Ready    worker   17h   v1.17.1
``` 

But when running the test on the default namespace I see -

```
$ oc create -f https://gist.githubusercontent.com/suhanime/683ee7b5a2f55c11e3a26a4223170582/raw/d893db98944bf615fccfe73e6e4fb19549a362a5/WinWebServer.yaml -n default
service/win-webserver created
deployment.apps/win-webserver created
```

But then -

```
$ oc describe pod/win-webserver-6f5bdc5b95-nl7rz
...
Events:
  Type     Reason                  Age                  From                  Message
  ----     ------                  ----                 ----                  -------
  Normal   Scheduled               116s                 default-scheduler     Successfully assigned default/win-webserver-6f5bdc5b95-nl7rz to win-worker1
  Warning  FailedCreatePodSandBox  114s                 kubelet, win-worker1  Failed to create pod sandbox: rpc error: code = Unknown desc = failed to set up sandbox container "d63e0d5ebef238519b4ac6329558db3eaefadfc0c5a7005afc8705b5986c0592" network for pod "win-webserver-6f5bdc5b95-nl7rz": networkPlugin cni failed to set up pod "win-webserver-6f5bdc5b95-nl7rz_default" network: error while GETHNSNewtorkByName(OVNKubernetesHybridOverlayNetwork): Network OVNKubernetesHybridOverlayNetwork not found
```

Any advice to diagnose and fix this ?  Thanks.

**Comment by tibcoplord** [comment]
I tried deleting the windows server and re-creating it via the wni program ... initially it got further but was unable to pull an image from imagestream ( host image-registry.openshift-image-registry.svc not found ).  But even so, after a re-start of the azure vm's the reported issue returned.

**Comment by tibcoplord** [comment]
Oh, if it matters, it took a couple of attempts to setup the windows server -

first pass fails with 

```
$ ansible-playbook -i hosts windows-machine-config-bootstrapper/tools/ansible/tasks/wsu/main.yaml -v
...
TASK [Wait for the hybrid overlay] ********************************************************************************************************************************************************************************************************************************************************************
FAILED - RETRYING: Wait for the hybrid overlay (12 retries left).
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: TypeError: Can't mix strings and bytes in path components
fatal: [52.149.234.71]: FAILED! => {"msg": "Unexpected failure during module execution.", "stdout": ""}
```

re-run -

```
$ ansible-playbook -i hosts windows-machine-config-bootstrapper/tools/ansible/tasks/wsu/main.yaml -v
...
PLAY RECAP ********************************************************************************************************************************************************************************************************************************************************************************************
52.149.234.71              : ok=33   changed=21   unreachable=0    failed=0    skipped=5    rescued=0    ignored=4   
localhost                  : ok=27   changed=24   unreachable=0    failed=0    skipped=3    rescued=0    ignored=0   
```

**Comment by tibcoplord** [comment]
Seen with ocp 4.4.4 and 4.4.7 ... this configuration doesn't survive a reboot.

**Comment by aravindhp** [comment]
The configuration is not expected to survive a reboot. You need to re-run the playbook which should succeed.

**Comment by tibcoplord** [comment]
Yes, re-running the playbook fixes the issue.  However its obviously not sustainable.

Do you know if there is an alternative OpenShift/RedHat recommended way to add windows nodes to an OpenShift cluster ?

Thanks.

**Comment by aravindhp** [comment]
I agree that this is not sustainable however this is just a [dev-preview](https://www.openshift.com/blog/announcing-the-developer-preview-of-windows-server-containers-in-openshift-4.4?hs_preview=JPkNLXPw-28747811980) feature to allow customers to try this out. The [long term future](https://github.com/openshift/enhancements/pull/364) is going to be an operator to enable Windows workloads.

**Comment by tibcoplord** [comment]
Ah, got it.  Thanks for the info.

**Comment by aravindhp** [comment]
/close

@tibcoplord please use [Windows Machine Config Operator](https://github.com/openshift/windows-machine-config-operator) that we just released as a community operator on OpenShift 4.6+.
